cypress/e2e/spec-html.cy.js

```diff
@@ -57,6 +57,20 @@ it("blocks external links", () => {
     .should("include.text", "An error has occurred");
 });
 
+it("allows permitted external links", () => {
+  localStorage.clear();
+  cy.visit(baseUrl);
+  cy.get("div[class=cm-content]").invoke(
+    "text",
+    '<a href="https://rpf.io/seefood">some external link</a>',
+  );
+  cy.get(".btn--run").click();
+  const externalLink = getIframeBody().find("a");
+  externalLink.click();
+  cy.url().should("be.equals", baseUrl);
+  cy.get("div[class=modal-content__header]").should("not.exist");
+});
```

Comment:
Cypress test now checks that the url hasn't changed (so we're definitely on the same page after the link is clicked) and that no modal appears. But: 
- Should we check to make sure it's not an error modal?
- Should we check that the iframe has some content that will confirm it's loaded the page? I'm reluctant to check for specific content since if the `seefood` project changes then that will break this test. (I don't think it's an issue if the `seefood` project is taken down completely since the link will still be permitted and will just render a missing page)

Reply:
I think what you've got it fine 👍